assets/app/js/renderer.js

```diff
@@ -48,16 +48,18 @@ $(document).ready(() => {
   const barChart = new AboutCodeBarChart('#tab-barchart', aboutCodeDB)
     .on('bar-clicked', (attribute, value) => {
       // Show files that contain attribute value selected by user in bar chart
-      if (value !== 'No Value Detected') {
         cluesTable.clearColumnFilters();
-        cluesTable.setColumnFilter(attribute, value);
-        
+        if (value !== 'No Value Detected') {
+          cluesTable.setColumnFilter(attribute, value);
+        } else {
+          cluesTable.setColumnFilter(attribute, 'about_code_data_table_no_value_detected');
```

Comment:
It would be better to do something like `cluesTable.setColumnFilter(attribute, cluesTable.NO_VALUE_DETECTED)`

I think that should work; if it does not, then the current solution is fine.